### PR TITLE
Add script for closing issues without response

### PR DESCRIPTION
**Pull request by adasiewiczr**
### 🎯 Goal

Add script for closing issues without response

### 🛠 Implementation details

Added a Github action for closing issues marked with `waiting for response` label that didn't receive any response from the author for at least 7 days.
If the original author comes back and gives more information, the label is removed and the issue is reopened, if necessary

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS (task created))
- [x] Reviewers added

### CODE REVIEW

File: .github/workflows/no-response.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
+        with:
+          token: ${{ github.token }}
+          daysUntilClose: 7
+          responseRequiredLabel: waiting for response
+          responseRequiredColor: D28960
```
Comment (zsmb13) [inline .github/workflows/no-response.yml:19]: Feels like we could just create the label on the UI, and then we don't need the hex code in the config here 😃

**Review by zsmb13 — APPROVED** [review]
LGTM, we can adjust it later if this turns out to be too aggressive, but really helpful otherwise

File: .github/workflows/no-response.yml (hunk 1 above)
Comment (adasiewiczr) [reply .github/workflows/no-response.yml:19]: The label is already created and you're right - we don't need it in the config

**Review by amitkma — APPROVED** [review]
LGTM

**Review by JcMinarro — APPROVED** [review]

**Pull request merged**